### DISCUSSION TITLE
Add docs

### DISCUSSION THREAD

**Discussion by clason** [tree-sitter-grammars/.github#7] [Ideas]
Separate `docs` repo with manuals:

* writing grammars (workflow, intro, advanced topics from amaanq)
* writing queries (intro, tools, tutorials?)
* ...?

Include content from https://github.com/sogaiu/ts-questions (with permission). (Current upstream docs are not great for queries: spread over two pages and mix in crate config. Seems easier to start from scratch.)

**Comment by ObserverOfTime** [comment]
Do we need a separate repo? They could be in this repo in a docs folder (or the wiki).

**Reply by clason** [reply]
For visibility (and possible deployment), I would say so, yes. (I envision this to be a growing effort.)

Doesn't mean we can't start smaller, of course.
